backend/src/ee/services/permission/org-permission.ts

```diff
@@ -28,6 +28,14 @@ export enum OrgPermissionKmipActions {
   Setup = "setup"
 }
 
+export enum OrgPermissionMachineIdentityAuthTemplateActions {
+  ListTemplates = "list-templates",
+  EditTemplates = "edit-templates",
+  CreateTemplates = "create-templates",
+  DeleteTemplates = "delete-templates",
+  UnlinkTemplates = "unlink-templates"
```

Comment:
wait.. when do we use this UnlinkTemplates` action? 

also it looks like we're missing the permission to allow somebody to use a template?